uber/site_sections/registration.py

```diff
@@ -55,13 +55,14 @@ def index(self, session, message='', page='0', search_text='', uploaded_id='', o
         attendees = attendees.order(order)
 
         groups = []
-        for group in session.query(Group) \
-                            .options(joinedload(Group.leader)) \
-                            .filter(Group.id.in_(
-                                session.query(Attendee.group_id)
-                                       .filter(Attendee.group_id != None, Attendee.first_name == '')
-                                       .distinct()
-                                       .subquery())).all():
+        for group in (session.query(Group)
+                             .options(joinedload(Group.leader))
+                             .filter(Group.status != c.WAITLISTED,
+                                     Group.id.in_(
+                                        session.query(Attendee.group_id)
+                                               .filter(Attendee.group_id != None, Attendee.first_name == '')
+                                               .distinct()
+                                               .subquery()))):
```

Comment:
> some of the Merchants will come ad attendees anyway, we wouldn't want to delete those badges.
￼
The problem is, those badges aren't valid as it stands now, because they're not paid and cannot be marked as paid without the dealer group being approved.  So those registrations are worthless as is.

Deleting them is what we've always done in the past.  I'll email Danielle tonight to ask whether she wants to do that this year as well.  We usually send out an automated email to the effect of "Only X slots opened up on our waitlist and we had hundreds of applications, so there's no room for you."  I'll ask whether that's also something she wants this year.